README.rst

```diff
@@ -153,20 +153,23 @@ Developers". Feel free to cite our work in your research using the following Bib
 
 .. code-block::
 
-    @software{uieda_leonardo_2020_4025418,
+    @software{pygmt_2021_4522136,
       author       = {Uieda, Leonardo and
                       Tian, Dongdong and
                       Leong, Wei Ji and
                       Toney, Liam and
+                      Schlitzer, William and
+                      Grund, Michael and
```

Comment:
Guten tag @michaelgrund, could you please provide your full affiliation (or would you prefer to use Unaffiliated)? Preferably in the format: Company, City, Country.

Better yet if you do a quick Pull Request to update it in https://github.com/GenericMappingTools/pygmt/blame/5faf06c30f2861e92e8bf8c7377451b95baeaec0/AUTHORS.md#L15. Just need it for the Zenodo DOI later.

Reply:
@weiji14 If you don't hear from him, his ORCID page lists "Innoplexia GmbH: Heidelberg, Baden-Württemberg, DE" as his affiliation

Reply:
Ok!

Reply:
@weiji14 I updated my affiliation, sorry for being late :sweat_smile: .